libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -189,6 +189,7 @@ class SRV_Channel {
         k_rcin15_mapped         = 154,
         k_rcin16_mapped         = 155,
         k_lift_release          = 156,
+        k_throttle_tilt         = 157,
```

Comment:
Should we enforce that users don't assign this to a actual output? It does not abide by safety state for example (or E-stop).

Reply:
i don't think that is necessary, for any particular vehicle type most of these are not available, and I didn't add it to the doc list, so won't appear in the dropdown options